packages/tooling/fast-vscode-syntax-highlighting/syntaxes/fast-tagged-templates.tmLanguage.json

```diff
@@ -15,7 +15,7 @@
     ],
     "repository": {
         "css-tagged-template": {
-            "begin": "(?i)(?<!\\.)(css|((?:\\/\\*\\s*?)css(?:\\s*\\*\\/)))\\s*(`)",
+            "begin": "(?i)(?<!\\.)((css(\\.)?(partial)?)|((?:\\/\\*\\s*?)(css(\\.)?(partial)?)(?:\\s*\\*\\/)))\\s*(`)",
```

Comment:
Adding capture groups (parentheses) changes the order of the groups used in the `beginCaptures` property. These changes add six new capture groups, which would push the `punctuation.definition.string.template.begin.ts` (which handles the backtick that starts the template literal) from capture 3 to capture 9.

Reply:
Modifying the capturing groups to be non-blocking can get it down to four. The `.` in `css.partial` needs to be captured in the first half to treat it as a property accessor, and I added a `\b` to ensure that we're only matching `css.partial` and not something like `SomeCoolLibraryForCSS.partial`.

```ts
            "begin": "(?i)(?<!\\.)(\\bcss(?:(\\.?)partial)|((?:\\/\\*\\s*?)css(?:\\.?partial)?(?:\\s*\\*\\/)))\\s*(`)",
            "beginCaptures": {
                "1": {
                    "name": "entity.name.function.tagged-template.ts"
                },
                "2": {
                    "name": "punctuation.accessor.ts"
                },
                "3": {
                    "name": "comment.block"
                },
                "4": {
                    "name": "punctuation.definition.string.template.begin.ts"
                }
            },
```